plugins/vpp/abfplugin/vppcalls/vpp1908/dump_abf_vppcalls.go

```diff
@@ -149,19 +148,28 @@ func (h *ABFVppHandler) dumpABFPolicy() ([]*vppcalls.ABFDetails, error) {
 	return abfs, nil
 }
 
-// Parses IP address to string. IP version is read from 'afi' where 1==IPv4 and 2==IPv6
-func parseNextHopToString(nh []byte, IPv uint8) string {
-	var nhIP net.IP = nh
-	if IPv == 1 {
+// returns next hop IP address
+func parseNextHopToString(nh abf.FibPathNh, proto abf.FibPathNhProto) string {
+	var nhIP net.IP = make([]byte, 16)
+	copy(nhIP[:], nh.Address.XXX_UnionData[:])
```

Comment:
You can avoid using `XXX_` field directly and the unnecessary preallocation of `net.IP` similarly to how we dump arps: https://github.com/ligato/vpp-agent/blob/2fa8373919019b61c7796f141b66f389182cac9b/plugins/vpp/l3plugin/vppcalls/vpp1908/arp_dump.go#L62-L69